pkgs/applications/kde/kdeconnect-kde.nix

```diff
@@ -44,39 +37,36 @@ mkDerivation rec {
   ];
 
   buildInputs = [
-    libfakekey
-    libXtst
-    qtmultimedia
-    qtgraphicaleffects
-    pulseaudio-qt
-    kpeoplevcard
-    kpeople
-    kirigami2
-    ki18n
-    kiconthemes
     kcmutils
     kconfigwidgets
     kdbusaddons
+    ki18n
+    kiconthemes
+    kio
+    kirigami2
     knotifications
+    kpeople
+    kpeoplevcard
+    kwayland
+    libXtst
+    libfakekey
+    pulseaudio-qt
     qca-qt5
+    qtgraphicaleffects
+    qtmultimedia
     qtx11extras
-    makeWrapper
-    kwayland
-    kio
   ];
 
-  nativeBuildInputs = [ extra-cmake-modules kdoctools ];
+  nativeBuildInputs = [ extra-cmake-modules kdoctools makeWrapper ];
 
   postInstall = ''
     wrapProgram $out/libexec/kdeconnectd --prefix PATH : ${lib.makeBinPath [ sshfs ]}
```

Comment:
Not strictly part of the issue, but since this expression is changed, perhaps it'd be worth sparing the double wrap I suspect happens here.

Reply:
Good point, done.